### ISSUE TITLE
Accessibility problem with "Follows You" meessage

### ISSUE THREAD

**Issue by mikehamca** [mastodon/mastodon#26133]
### Steps to reproduce the problem

I looked at a user profile and saw a dark-colored rectangle at the upper left. For me (colorblind), it was totally unreadable: black letters on dark-red background. By copying the block and working on the colors, I realized it said: "Follows You." 

The color scheme is new — previously the message was readable — and the selection of colors did not take accessibility into account.  

I highly recommend this document: https://venngage.com/blog/accessible-colors/

### Expected behaviour

The message should be present in a color combination that all can see.

### Actual behaviour

Message was present in a color combination unreadable by those who are colorblind.

### Detailed description

Black letters on a dark-red background (or dark-red letters on a black background) are unreadable to many who are colorblind. For accessibility, develops should choose color combinations that are legible to most users. See: https://venngage.com/blog/accessible-colors/

### Mastodon instance

mstdn.ca 

### Mastodon version

v4.1.2

### Browser name and version

Vivaldi 6.1.3035.204 (Stable channel) (arm64)

### Operating system

macOS 13.4.1

**Comment by renchap** [comment]
What is your instance?

On the latest Mastodon version, the text color is white on a dark background, which provides adequate contrast.

<img width="149" alt="image" src="https://github.com/mastodon/mastodon/assets/42070/1ed46231-36db-4658-9f34-4afdbc2e4eec">

**Comment by mikehamca** [comment]
Apologies. Meant to include that. It is mstdn.ca <http://mstdn.ca/> 

Michael

**Comment by ChadOhman** [comment]
Hello, we do not have a custom theme or custom CSS set up on the instance.

**Comment by ClearlyClaire** [comment]
Hi! The contrast is definitely not always ideal, but I'm not able to find any occurrence of black letters on dark background here.

Do you have a screenshot? Which theme are you using?

**Comment by mikehamca** [comment]
Here is a screenshot:

￼

I am using Vanilla Mastodon in Mastodon (Light) and the Advanced Web Interface.

Here’s a larger screenshot:

…. Well, this is weird. I went to take a larger screenshot, and now the “Follows You” tag does not appear at all, even on those accounts that do follow me. 

I don’t understand why “Follows You” is missing, but as you can see, when I had it, it had gone bad. (At one time, it was perfectly legible.)

Michael

**Comment by ClearlyClaire** [comment]
Sorry, but your message has a unicode replacement character instead of a screenshot. If you're using the vanilla Mastodon (light), then this sounds like it has been fixed in #25993

**Comment by liza** [comment]
just wanna say, first of all, i cried after editing my first toot. yes, i shed tears because as a dyslexic writer, illegible posts were the bane of my existence. so THANK YOU for that edit function.

now on to OPs point. 

am dyslexic and have "text blindness" (dysgraphia). i just found out a few days ago this is how you flag followers. 

![image](https://github.com/mastodon/mastodon/assets/185027/e8d4b16e-fb13-4b91-b23e-06cd8d1ce445)

even though it may look okayish when zoomed in, in a busy page, this won't register for people with color blindness, dysgraphia, aphantasia, aka the neuroatypicals. 

that flag should be right by the follow button. it's a little tweak; but it would be a huge usability improvement. 

am on the docs portal and  it may be my dysgraphia, but am not seeing any info on theme development. what theme engine are you using?

**Comment by mikehamca** [comment]
Here's a screenshot of how the "Follows You" message looks on my Macbook Air.
<img width="340" alt="Screenshot 2023-07-25 at 7 35 48 PM" src="https://github.com/mastodon/mastodon/assets/117941733/63972fea-b647-4ef5-9af1-25690b74ab34">

**Comment by gunchleoc** [comment]
The layout got already changed in version 4.2.0

![contrast-4 2 0](https://github.com/mastodon/mastodon/assets/4095570/a042e6f2-442e-481d-bd56-b96abb1d33ed)

It looks OK to me too in 4.1.6, although the colors are reversed there:

![contrast-4 1 6](https://github.com/mastodon/mastodon/assets/4095570/55fc4e9f-1a16-4d3f-b06d-2a0f4b7d97d7)

Have you tried clearing our browser's CSS cache/doing a forced refresh of the page? In Firefox, that would be the keyboard shortcut Ctrl+F5, I don't know how to do that in Safari - seems to be non-trivial https://superuser.com/questions/186594/how-can-i-force-safari-to-perform-a-full-page-reload-without-using-the-mouse

**Comment by diondiondion** [comment]
As of Mastodon 4.6, the "Follows you" badge was moved next to the account's name and should be much more readable.

<img width="361" height="228" alt="Image" src="https://github.com/user-attachments/assets/27de6ea7-5a93-4b25-9b55-a1529a8fff5e" />
